File: src/assets/usage-examples-code/windows/window_toggle_fullscreen_named/window_toggle_fullscreen_named.cs
using System;
using SplashKitSDK;

class Program
{
    static void Main()
    {
        // Open a window initially
        Window myWindow = new Window("My Window", 800, 600);

        // Wait for a short time
        SplashKit.Delay(2000);

        // Toggle fullscreen mode
        SplashKit.WindowToggleFullscreen("My Window");

        // Keep the window open until manually closed
        while (!SplashKit.WindowCloseRequested("My Window"))
        {
            SplashKit.ProcessEvents();
            SplashKit.Delay(100);
        }
    }
}

File: src/assets/usage-examples-code/windows/window_toggle_fullscreen_named/window_toggle_fullscreen_named.cpp
#include "splashkit.h"

int main()
{
    // Open a window initially
    window myWindow = open_window("My Window", 800, 600);

    // Wait for a short time
    delay(2000);

    // Toggle fullscreen mode
    window_toggle_fullscreen("My Window");

    // Keep the window open until manually closed
    while (!window_close_requested(myWindow))
    {
        process_events();
        delay(100);
    }

    return 0;
}
